services/php-apache/Dockerfile

```diff
@@ -1,5 +1,7 @@
-ENV COMPOSER_NO_INTERACTION=1 \
-    DOCROOT=/var/www/html
+ENV COMPOSER_DISCARD_CHANGES=true
+ENV COMPOSER_NO_INTERACTION=true
```

Comment:
Interesting - you usually shouldn't need this unless the terminal is interactive. What happens if someone cli's in and runs composer?

Reply:
We can remove it, however it was already here. :)

Reply:
It's fine, I saw you changing 1 to true so figured knew something more.

Reply:
What do you think @bchavet ? Should we remove the COMPOSER_NO_INTERACTION environment variable?

Reply:
The docs say this variable is intended for CI/CD, and it's been here for quite a while. I think we should keep it to avoid any unexpected build changes, if nothing else